packages/backend-core/src/api/collection/OrganizationApi.ts

```diff
@@ -3,17 +3,50 @@ import { AbstractApi } from './AbstractApi';
 
 const basePath = '/organizations';
 
+type OrganizationMetadataParams = {
+  publicMetadata?: Record<string, unknown>;
+  privateMetadata?: Record<string, unknown>;
+};
+
 type CreateParams = {
   name: string;
   createdBy: string;
+} & OrganizationMetadataParams;
+
+type OrganizationMetadataRequestBody = {
+  publicMetadata?: string;
+  privateMetadata?: string;
 };
 
 export class OrganizationApi extends AbstractApi {
   public async createOrganization(params: CreateParams) {
+    const { publicMetadata, privateMetadata } = params;
     return this._restClient.makeRequest<Organization>({
       method: 'POST',
       path: basePath,
-      bodyParams: params,
+      bodyParams: {
+        ...params,
+        ...stringifyMetadataParams({
+          publicMetadata,
+          privateMetadata,
+        }),
```

Comment:
I'm not sure if we need to stringify the metadata parameters here, but I followed what we already do for invitation and user metadata.